modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2376,17 +2376,21 @@ def explode(self, column):
     # UDF (apply and agg) methods
     # There is a wide range of behaviors that are supported, so a lot of the
     # logic can get a bit convoluted.
-    def apply(self, func, axis, *args, **kwargs):
+    def apply(self, func, axis, is_series=False, *args, **kwargs):
```

Comment:
Yeah, I was wondering if there is a better place to encapsulate this logic. It might be better to just hide this in the `kwargs` as well, but I'm not too sure. 

I'm not sure why there isn't a docstring here, but we can add one.